rpm/debugedit.spec

```diff
@@ -23,10 +23,6 @@ Requires: elfutils
 Requires: binutils, gawk, coreutils, xz
 # For find and xargs
 Requires: findutils
-# For do_file, gdb_add_index
-# We only need gdb-add-index, so suggest gdb-minimal (full gdb is also ok)
-Requires: /usr/bin/gdb-add-index
-Suggests: gdb-minimal
```

Comment:
I don't think we should to that it slows down debugging.

Reply:
Correct, but this PR is for packaging `debugedit` for the purposes of updating `rpm`. This PR is an improvement of the current situation already, so instead of pulling `gdb` into the mix let's pull this in and improve the feature set later.

Reply:
How does this pull gdb into the mix, where there changes to gdb needed to pull this off?

I any case when we cut corners we should document that. In this instance I would keep the lines above in the spec commented out with a FIXME.